style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
- [ ] Kindly instead of using the `h1` sector to style your header, use an `id` or `class` instead, add the class name or id name in your HTML to your title, then call it in the CSS file by the corresponding selector.